mppsolar/protocols/pi18.py

```diff
@@ -506,7 +610,8 @@ def __init__(self, *args, **kwargs) -> None:
             "MCHGV",
             "MUCHGC",
         ]
-        self.DEFAULT_COMMAND = "PI"
+        self.DEFAULT_COMMAND = ["PIRI"]
+        self.ID_COMMANDS = ["ID"]
```

Comment:
this needs to be PI only, so getDeviceId is the same for all devices that use this protocol.

Reply:
ahh ok, is put back on Pi